apps/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -649,9 +644,14 @@ export abstract class BaseInstallManager {
 
   private _syncTempShrinkwrap(shrinkwrapFile: BaseShrinkwrapFile | undefined): void {
     if (shrinkwrapFile) {
-      // If we have a (possibly incomplete) shrinkwrap file, save it as the temporary file.
-      shrinkwrapFile.save(this.rushConfiguration.tempShrinkwrapFilename);
-      shrinkwrapFile.save(this.rushConfiguration.tempShrinkwrapPreinstallFilename);
+      Utilities.syncFile(
+        this._rushConfiguration.getCommittedShrinkwrapFilename(this.options.variant),
+        this.rushConfiguration.tempShrinkwrapFilename
+      );
+      Utilities.syncFile(
+        this._rushConfiguration.getCommittedShrinkwrapFilename(this.options.variant),
+        this.rushConfiguration.tempShrinkwrapPreinstallFilename
+      );
```

Comment:
When I invoke `rush install` using this PR, for the already-up-to-date state (where `pnpm install` is skipped), the console output looks like this:

```
Found files in the "common/git-hooks" folder.
Successfully installed these Git hook scripts: pre-commit

Trying to acquire lock for pnpm-5.18.9
Acquired lock for pnpm-5.18.9
Found pnpm version 5.18.9 in C:\Users\Owner\.rush\node-v12.20.1\pnpm-5.18.9

Symlinking "C:\Git\MyRepo\common\temp\pnpm-local"
  --> "C:\Users\Owner\.rush\node-v12.20.1\pnpm-5.18.9"
Copying C:\Git\MyRepo\common\config\rush\.npmrc --> C:\Git\MyRepo\common\temp\.npmrc
Updating C:\Git\MyRepo\common\temp\pnpmfile.js

Updating workspace files in C:\Git\MyRepo\common\temp
Finished creating workspace (0.16 seconds)
Updating C:\Git\MyRepo\common\temp\pnpm-lock.yaml
Updating C:\Git\MyRepo\common\temp\pnpm-lock-preinstall.yaml

Checking node_modules in C:\Git\MyRepo\common\temp


Rush install finished successfully. (0.58 seconds)
```

(The messages about `pnpm-lock.yaml` and `pnpm-lock-preinstall.yaml` are new in this PR.)

Some observations:
- `Symlinking` shows the destination on a separate line from the source
- `Copying` shows the destination on the same line as the source.  (When I looked at the code, it seems that actually `Utilities.copyAndTrimNpmrcFile()` is not actually a simple "copy" operation, but rather transforms the file based on environment variables.)
- `Updating` shows ONLY the destination.  If the source file does not exist, then `Utilities.syncFile()` will instead say `Deleting ${destinationPath}`.  (But this code path seems to be impossible for  `pnpm-lock.yaml`.)
- Some other common/temp files don't get mentioned at all: `clientPnpmfile.js`, `current-variant.json`, `pnpmfileSettings.json`, `pnpm-workspace.yaml`
- The `Checking node_modules in` message is actually testing whether we can skip `pnpm install` or not.

🤔 Maybe this is fine?  Some possible improvements:

1. Change `Utilities.copyAndTrimNpmrcFile()` to say "Transforming" and print the `-->` on a separate line (like how  "Symlinking" does)
2. Change `Utilities.syncFile()` to say "Copying" since it is an actual file copy.  The source is unimportant.
3. If `Checking node_modules in` determines to skip `pnpm install`,  maybe we should print a message like `Installation is already up to date.`

Reply:
> 3\. We should print a newline before `Updating C:\Git\MyRepo\common\temp\pnpm-lock.yaml` since it is unrelated to `Finished creating workspace (0.16 seconds)`

(Alternatively, maybe we should eliminate the message `Finished creating workspace (0.16 seconds)` -- is it important to measure this?  Have you ever seen it take a nontrivial amount of time?)

Reply:
I think I got all of these but you may want to try it out locally to see how it feels to you (but heads up my remote URL changed),